packages/trader/src/AppV2/Components/TradeParameters/trade-parameters.tsx

```diff
@@ -29,28 +31,40 @@ const TradeParameters = observer(({ is_minimized }: TTradeParametersProps) => {
     };
```

Comment:
@henry-deriv , while looking into this code, I felt there is a possibility of improvement. isVisible('someText') calls getTradeParams(symbol)[contract_type].includes looks we loop an array. So for every call isVisible once it loops array. What about we write a function in which we reduce getTradeParams(symbol)[contract_type] array to object. so very time isVisible is called it will just look for key in object, so we save on loop(includes). see if possible or I missed something